parts/linux/cloud-init/artifacts/nbcparser.sh

```diff
@@ -0,0 +1,9 @@
+#!/bin/bash
+set -x
+# TODO: propagate exit status to bootstrapper
```

Comment:
I think we will eventually have a boiler-plate CSE script which would run a script to poll the status of this unit and return an exit code based on the corresponding status - that'll be the only way we can get exit code information back to the caller I believe

Reply:
Yep, will add this in the next PR